mne/preprocessing/_css.py

```diff
@@ -0,0 +1,174 @@
+    filter_data : np.ndarray of float, shape (n_grad_sensors,n_times)
+        The data to be filtered, typically the magnetometer or EEG data.
+    r : int
+        The number of projection vectors.
+
+    Returns
+    -------
+    filtered_data : np.ndarray of int, shape (n_features)
+        The outlier indices.
+    """
+    # Orthonormalize gradiometer and magnetometer data by a QR decomposition
+    Qg, rg = np.linalg.qr(grad_data.T)
+    Qm, rm = np.linalg.qr(mag_data.T)
+
+    # Calculate cross-correlation
+    C = np.dot(Qg.T, Qm)
+
+    # Channel weights for common temporal subspace by SVD of cross-correlation
+    Y, S, Z = np.linalg.svd(C)
+
+    # Get temporal signals from channel weights
+    u = np.dot(Qg, Y)
+
+    # Project out common subspace (r is number of projection vectors)
+    filtered_data = filter_data
+    for i in range(r):
+        proj_vec = u[:, i].reshape(u.shape[0], 1)
+        NpProj = np.dot(filtered_data, proj_vec)
+        filtered_data = filtered_data - np.dot(NpProj, proj_vec.T)
+
+    return filtered_data
+
+
+def _plot_psd(evoked, evoked_subcortical, filt_inds, title):
+    """Plots the total power spectral density of data in evoked and
+    evoked_subcortical.
+
+    Plots the sum of the power spectral density across channels filt_inds.
+
+    Parameters
+    ----------
+    evoked : evoked object
+        Unprocessed data.
+    evoked_subcortical : evoked object
+        Processed data.
+    filt_inds : np.ndarray of int, shape (n_sensors)
+        The indicies of the data that were filtered.
+    title : string
+        The title of the plot
+    """
+    def getFFT(y, Fs):
+        Ft = np.fft.fft(y)
+        Ft = np.abs(Ft[0:int(len(Ft) / 2 + 1)])**2
+        Y = np.sqrt(Ft / (len(y) * Fs))
+        f = Fs / 2 * np.linspace(0, 1, int(len(y) / 2 + 1))
+        return [f, Y]
+
+    def sum_fft(evoked, evoked_subcortical, mag_ind, chs):
+        fft_length = int(evoked.times.shape[0] / 2 + 1)
+        fft_unprocessed = np.zeros((fft_length))
+        fft_processed = np.zeros((fft_length))
+        for ch in chs:
+            signal_unprocessed = evoked.data[mag_ind[ch], :]
+            signal_processed = evoked_subcortical.data[mag_ind[ch], :]
+            fft_unprocessed = fft_unprocessed + \
+                getFFT(signal_unprocessed, evoked.info['sfreq'])[1]
+            f, fft_processed = fft_processed + \
+                getFFT(signal_processed, evoked.info['sfreq'])
+
+        return fft_unprocessed, fft_processed, f
+
+    fft_unprocessed, fft_processed, f = \
+        sum_fft(evoked, evoked_subcortical, filt_inds,
+                chs=range(len(filt_inds)))
+    plt.figure()
+    plt.plot(f, fft_unprocessed, label='raw data')
+    plt.plot(f, fft_processed, label='processed')
+    plt.title(title)
+    plt.xlabel('Frequency (Hz)')
+    plt.ylabel('Spectral density')
+    plt.legend()
+    plt.tight_layout()
+    plt.show()
+
+
+def filter_cortical(evoked, projection_vectors=6, show_psd=False):
```

Comment:
projection_vectors -> n_proj?

Reply:
also plot should be separate. All plotting functions return matplotlib fig handle

Reply:
+1 for n_proj

also I would rename this to `cortical_signal_suppression`